src/main/java/appeng/core/sync/packets/PacketMockExplosion.java

```diff
@@ -20,32 +20,26 @@
 
 
 import io.netty.buffer.ByteBuf;
-import io.netty.buffer.Unpooled;
 
-import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.world.World;
 
-import cpw.mods.fml.relauncher.Side;
-import cpw.mods.fml.relauncher.SideOnly;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
 
 import appeng.core.CommonHelper;
 import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.network.INetworkInfo;
+import appeng.core.sync.AppEngPacketHandler;
 
 
-public class PacketMockExplosion extends AppEngPacket
+public class PacketMockExplosion implements AppEngPacket, AppEngPacketHandler<PacketMockExplosion, AppEngPacket>
 {
 
-	public final double x;
-	public final double y;
-	public final double z;
+	private double x;
+	private double y;
+	private double z;
 
 	// automatic.
-	public PacketMockExplosion( ByteBuf stream )
+	public PacketMockExplosion()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)